db/migrate/20151212182619_add_columns_to_adoption_application.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnsToAdoptionApplication < ActiveRecord::Migration
+  def change
+    add_column :adoption_applications, :current_rats, :string
+    add_column :adoption_applications, :previous_rats, :string
+  end
```

Comment:
Since this is still a PR, what are your thoughts on modifying the original migration to include these?